src/routes/post-article.ts

```diff
@@ -0,0 +1,22 @@
+import { constants } from 'http2';
+import { Context, Middleware, Next } from 'koa';
+import { JsonLdObj } from 'jsonld/jsonld-spec';
+import uniqueString from 'unique-string';
+import createHttpError from 'http-errors';
+import Articles from '../articles';
+
+export default (articles: Articles): Middleware => (
+  async ({ request, response }: Context, next: Next): Promise<void> => {
+    const article = request.body as JsonLdObj;
```

Comment:
Really we'll want the context to know that the body is JSON-LD. We can follow this up.